pkg/util/interrupt/interrupt.go

```diff
@@ -35,6 +42,10 @@ func Chain(handler *Handler, notify ...func()) *Handler {
 	return New(handler.Signal, append(notify, handler.Close)...)
 }
 
+// New creates a new handler that guarantees all notify functions are run after the critical
+// section exits (or is interrupted by the OS), then invokes the final handler. If no final
+// handler is specified, the default final is `os.Exit(0)`. A handler can only be used for
```

Comment:
Perhaps `os.Exit(2)` would be better, since that's the default exit code that the Go runtime uses for the signals we're catching?
Exiting with 0 tells that the process exited under "normal", okay, circumstances, which is not the case here?